### PR TITLE
Add more options to the "lvs" spec

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### CODE REVIEW

File: insights/specs/insights_archive.py
Comment (xiangce) [inline insights/specs/insights_archive.py:134]: Please move the new command to the first place of the list.

**Review by xiangce — APPROVED** [review]
Approved pending the approval of spec

**Review by bfahr — APPROVED** [review]

**Pull request merged**
